tests/ops/fixtures/saas/firebase_auth_fixtures.py

```diff
@@ -114,6 +115,7 @@ def firebase_auth_user(firebase_auth_secrets) -> Generator:
         email_verified=False,
         display_name="John Doe",
         photo_url="http://www.example.com/12345678/photo.png",
+        phone_number="+1" + str(randint(1000000000, 9999999999)),
```

Comment:
I'm guessing there's no validation on the phone number other than it has to be unique within the system?

Reply:
not that has come up yet in testing, at least :)

i'm a little bit unsure even on the uniqueness constraints, i.e. if the user has both an `email` and a `phone_number`, i'm not even sure if `phone_number` must be unique. but i'm guessing it probably is?